Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends apt-utils dialo
 
 # Add curl, bzip2 and some dev libs
 RUN apt-get update -qq && apt-get -y --no-install-recommends install \
-    curl \
```

Comment:
If curl is not there to start, why take it out? Could be useful!

Reply:
Okay, I was figuring if we didn't use it for downloading the data, then we probably don't need it. But I'll add it back.